release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1703,6 +1703,10 @@ Kubernetes 1.29 removed the following deprecated APIs, so you must migrate manif
 [id="ocp-4-16-builds-bug-fixes_{context}"]
 ==== Builds
 
+* Previously, clusters that updated from earlier versions to 4.16 continued to allow builds to be triggered by unauthenticated webhooks. With this release, new clusters require build webhooks to be authenticated. Builds are not triggered by unauthenticated webhooks unless a cluster administrator allows unauthenticated webhooks in the namespace or cluster. (link:https://issues.redhat.com/browse/OCPBUGS-33378[*OCPBUGS-33378*])
+
+* Previously, if the developer or cluster administrator used lowercase environment variable names for proxy information, these environment variables were carried into the build output container image. At runtime, the proxy settings were active and had to be unset. With this release, lowercase versions of the `\*_PROXY environment` variables are prevented from leaking into built container images. Now, `buildDefaults` are only kept during the build and settings created for the build process only are removed before pushing the image in the registry. (link:https://issues.redhat.com/browse/OCPBUGS-34825[*OCPBUGS-34825*])
```

Comment:
No idea why the link text is not rendering correctly. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/39cbb30e-1753-4060-96bd-9256285542a0)

Reply:
It looks fine in the preview on VSCODE. Bizarre.

Reply:
@mburke5678 FYI it was the escaped `*` messing it up. Alex figured out a fix with a passthrough instead of an escape :nerd_face:

Reply:
@cbippley Wow. The escape worked all the way from back there! Nice catch team.

Reply:
@mburke5678 Jeana and Alex figured it out!